src/app/modules/main/chat_section/chat_content/messages/controller.nim

```diff
@@ -52,6 +52,12 @@ method init*(self: Controller) =
       return
     self.delegate.newMessagesLoaded(args.messages, args.reactions, args.pinnedMessages)
 
+  self.events.on(SignalType.Message.event) do(e: Args):
```

Comment:
@jrainville since `SignalType.Message.event` is a global signal, coming from `status-go` directly, it shouldn't be handled in modules, but in appropriate service and from there appropriate signal should be sent which one or many modules may be connected to. 

For a reference please check `proc doConnect(self: Service) =` from the contact service (`statusUpdates` and `contacts` properties from the `MessageSignal` are handled there) and apply in this PR something similar. Actually you just need to move this connection to the message service and emit a new signal from there which will be handled here.